src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -363,7 +363,7 @@ void NotationViewInputController::mousePressEvent(QMouseEvent* event)
     }
 
     if (viewInteraction()->isTextEditingStarted()) {
-        if (!hitElement || !hitElement->isText()) {
+        if (!hitElement || !hitElement->isTextBase()) {
```

Comment:
meant mouse navigation while editing most types of text elements wasn't working (https://github.com/musescore/MuseScore/issues/8812)